src/F2F.ReactiveNavigation.WPF/IMenuBuilder.cs

```diff
@@ -12,5 +12,7 @@ public interface IMenuBuilder
 		void AddMenuItem(IMenuCommand command);
 
 		void AddMenuItem(string header, Action action);
+
+		void AddMenuItems(string header, IEnumerable<IMenuCommand> commands);
```

Comment:
Can you please also add overrides that allows to specify an already existing menu item as a parent.
Like so:
void AddMenuItems(IMenuCommand parent, IEnumerable<IMenuCommand> commands); 
void AddMenuItems(IMenuCommand parent, IMenuCommand command);